utils/k8s_lib_injection/k8s_weblog.py

```diff
@@ -128,6 +128,9 @@ def install_weblog_pod_without_admission_controller(self, use_uds):
         )
         pod_body.spec.init_containers.append(init_container1)
         pod_body.spec.containers[0].env.append(client.V1EnvVar(name="DD_LOGS_INJECTION", value="true"))
+        pod_body.spec.containers[0].env.append(client.V1EnvVar(name="DD_IAST_ENABLED", value="true"))
+        pod_body.spec.containers[0].env.append(client.V1EnvVar(name="DD_APPSEC_SCA_ENABLED", value="true"))
+        pod_body.spec.containers[0].env.append(client.V1EnvVar(name="DD_APPSEC_ENABLED", value="true"))
```

Comment:
These are env vars to configure the injection, when we are not using the admission controller